mitosheet/src/mito/Mito.tsx

```diff
@@ -1073,13 +1076,6 @@ export const Mito = (props: MitoProps): JSX.Element => {
                             {getCurrOpenTaskpane()}
                         </div>
                     }
-                    {uiState.currOpenSearch.isOpen &&
-                        <SearchBar
-                            uiState={uiState}
-                            setUIState={setUIState}
-                            mitoAPI={mitoAPI}
-                        />
-                    }
                 </div>
```

Comment:
Hmm, where do we display it?

Reply:
I moved it to `EndoGrid.tsx` so that I could access the container div for the calls to `ensureCellIsVisible`.